tests/gui/control/test_progress.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from taipy.gui import Gui
+
+
+def test_circular_progress_indeterminate_md(gui: Gui, helpers):
+    md_string = "<|progress|>"
+    expected_list = ["<Progress"]
+    helpers.test_control_md(gui, md_string, expected_list)
+
+
+def test_circular_progress_determinate_md(gui: Gui, helpers):
+    md_string = "<|progress|value=50|showValue=True|>"
```

Comment:
this should be show_value

Reply:
Actually, when I'm changing it to `show_value` it's showing an error. The element which is being rendered still contains the `showValue` praram instead of `show_value`. I've changed it in the `factory.py` as well. <br>
```
AssertionError: show_value={true} not in <div class="md-para" key="div.0">
               <Progress key="Progress.0" libClassName="taipy-progress" showValue={true} value="50"></Progress>
               </div>```

Reply:
The python component should specify `show_value`
The generated component should have `showValue`
The change is made automatically by Taipy

Reply:
Okayy, so in the `md_string`, I've to change it to `show_value` and in the `expected_list`, I've to keep it as `showValue`, am I right?